src/main/java/org/mule/extension/db/internal/result/row/InsensitiveMapRowHandler.java

```diff
@@ -45,4 +50,17 @@
 
     return result;
   }
+
+  //TODO MULE-14614 - All these handlers should return TypedValues indicating their MimeType.
```

Comment:
hackit now so DB returns the correct types please. Leave the TODO to remove the hack